ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -77,12 +77,27 @@ export default {
         },
         iconOnly () {
             return this.props.icon && !this.props.units
+        },
+        gaugeStyle () {
+            return this.props.gstyle
+        },
+        updateOn () {
+            return `${this.segments} ${this.gaugeStyle} ${this.min} ${this.max}`
+        },
+        min () {
+            return this.props.min
+        },
+        max () {
+            return this.props.max
         }
     },
     watch: {
-        value: function (val, oldVal) {
+        value: function (val) {
             this.resize()
             this.update(val)
+        },
+        updateOn () {
+            this.update(this.value)
```

Comment:
watching for changes as https://github.com/FlowFuse/node-red-dashboard/pull/1238/files#r1732422011 and update the gauge ui